### PR TITLE
7k aggregator volume statistic

### PR DESCRIPTION
**Pull request by tuanh2000**
7k is an aggregator build on SUI blockchain

### CODE REVIEW

**Comment by llamatester** [comment]
The 7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators 7k-aggregator
    
    🦙 Running 7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    
    
    
    ERROR: Use Adapter v2 format for new adapters

**Comment by tuanh2000** [comment]
what should I do? I don't know what this error means.

File: aggregators/7k-aggregator/index.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	);
+
+	return {
+		dailyVolume,
+		timestamp,
+	};
+};
+
+const adapter: SimpleAdapter = {
+	adapter: {
```
Comment (dtmkeng) [inline aggregators/7k-aggregator/index.ts:16]: add `version: 2` above `adapter`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by tuanh2000** [comment]
Maybe we got another error, why timestamp is an object bro?

**Comment by tuanh2000** [comment]
oh, I know what happened here, let me fix it

**Comment by llamatester** [comment]
The 7k-aggregator adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators 7k-aggregator
    
    🦙 Running 7K-AGGREGATOR adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 07 Jul 2024 14:16:23 GMT
    End Date:	Mon, 08 Jul 2024 14:16:23 GMT
    ---------------------------------------------------
    
    SUI 👇
    Backfill start time: 28/6/2024
    NO METHODOLOGY SPECIFIED
    Daily volume: 1
    End timestamp: 1720448182 (2024-07-08T14:16:22.000Z)

**Comment by tuanh2000** [comment]
https://1drv.ms/f/c/4f8f7df9b045821e/Epbf9TKmQbZGsuviSWDDuE0Bkwk_P5tXt76mIG_sIYlt3w?e=oFFp8L
This is out logo, let's check the request one last time, I think it worked.

**Comment by tuanh2000** [comment]
website: https://7k.ag
twitter: https://twitter.com/7k_ag_

**Pull request merged**